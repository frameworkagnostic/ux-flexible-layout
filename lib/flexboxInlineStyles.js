'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
var flexboxInlineStyles = exports.flexboxInlineStyles = {
  container: {
    boxSizing: 'border-box',
    display: 'flex',
    flexWrap: 'wrap',
    width: '100%'
  },
  flex: {
    display: 'flex'
  },
  box: {
    boxSizing: 'border-box',
    margin: '0',
    padding: '0'
  },
  item: {
    boxSizing: 'border-box',
    margin: '0'
  },
  zeroMinWidth: {
    minWidth: 0
  },
  'direction-column': {
    flexDirection: 'column'
  },
  'direction-column-reverse': {
    flexDirection: 'column-reverse'
  },
  'direction-row-reverse': {
    flexDirection: 'row-reverse'
  },
  'wrap-nowrap': {
    flexWrap: 'nowrap'
  },
  'wrap-reverse': {
    flexWrap: 'wrap-reverse'
  },
  'align-self-center': {
    alignSelf: 'center'
  },
  'align-items-center': {
    alignItems: 'center'
  },
  'align-items-flex-start': {
    alignItems: 'flex-start'
  },
  'align-items-flex-end': {
    alignItems: 'flex-end'
  },
  'align-items-baseline': {
    alignItems: 'baseline'
  },
  'align-content-center': {
    alignContent: 'center'
  },
  'align-content-flex-start': {
    alignContent: 'flex-start'
  },
  'align-content-flex-end': {
    alignContent: 'flex-end'
  },
  'align-content-space-between': {
    alignContent: 'space-between'
  },
  'align-content-space-around': {
    alignContent: 'space-around'
  },
  'justify-xs-center': {
    justifyContent: 'center'
  },
  'justify-xs-flex-end': {
    justifyContent: 'flex-end'
  },
  'justify-xs-space-between': {
    justifyContent: 'space-between'
  },
  'justify-xs-space-around': {
    justifyContent: 'space-around'
  }
};
//# sourceMappingURL=flexboxInlineStyles.js.map