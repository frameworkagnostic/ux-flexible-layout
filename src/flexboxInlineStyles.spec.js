

import { flexboxInlineStyles } from './flexboxInlineStyles';

describe('flexbox inline styles', () => {
  test('isObject', () => expect(flexboxInlineStyles).toEqual(expect.any(Object)));
  test('snapshot', () => expect(flexboxInlineStyles).toMatchSnapshot());
});
