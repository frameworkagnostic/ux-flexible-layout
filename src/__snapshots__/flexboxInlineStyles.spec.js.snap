// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`flexbox inline styles snapshot 1`] = `
Object {
  "align-content-center": Object {
    "alignContent": "center",
  },
  "align-content-flex-end": Object {
    "alignContent": "flex-end",
  },
  "align-content-flex-start": Object {
    "alignContent": "flex-start",
  },
  "align-content-space-around": Object {
    "alignContent": "space-around",
  },
  "align-content-space-between": Object {
    "alignContent": "space-between",
  },
  "align-items-baseline": Object {
    "alignItems": "baseline",
  },
  "align-items-center": Object {
    "alignItems": "center",
  },
  "align-items-flex-end": Object {
    "alignItems": "flex-end",
  },
  "align-items-flex-start": Object {
    "alignItems": "flex-start",
  },
  "align-self-center": Object {
    "alignSelf": "center",
  },
  "box": Object {
    "boxSizing": "border-box",
    "margin": "0",
    "padding": "0",
  },
  "container": Object {
    "boxSizing": "border-box",
    "display": "flex",
    "flexWrap": "wrap",
    "width": "100%",
  },
  "direction-column": Object {
    "flexDirection": "column",
  },
  "direction-column-reverse": Object {
    "flexDirection": "column-reverse",
  },
  "direction-row-reverse": Object {
    "flexDirection": "row-reverse",
  },
  "flex": Object {
    "display": "flex",
  },
  "item": Object {
    "boxSizing": "border-box",
    "margin": "0",
  },
  "justify-xs-center": Object {
    "justifyContent": "center",
  },
  "justify-xs-flex-end": Object {
    "justifyContent": "flex-end",
  },
  "justify-xs-space-around": Object {
    "justifyContent": "space-around",
  },
  "justify-xs-space-between": Object {
    "justifyContent": "space-between",
  },
  "wrap-nowrap": Object {
    "flexWrap": "nowrap",
  },
  "wrap-reverse": Object {
    "flexWrap": "wrap-reverse",
  },
  "zeroMinWidth": Object {
    "minWidth": 0,
  },
}
`;
